cookbooks/aws-parallelcluster-platform/test/controls/dcv_spec.rb

```diff
@@ -219,84 +229,96 @@
 end
 
 control 'tag:config_dcv_services_correctly_configured' do
-  only_if do
```

Comment:
`only_if` is the suggested way to handle exclusions. My understanding is because it surfaces exclusions in the test report. If we have different test parts to be run under different conditions, it might be worth splitting the control.

Reply:
And I think this control should be actually split because it does too many checks.

Reply:
Done a first split (services vs files)